tests/api/driver_core.c

```diff
@@ -149,23 +179,32 @@ void test_fetch_txn_with_uuid(void) {
   TEST_ASSERT_EQUAL_INT32(SC_OK, broadcast_buffered_txn(&ta_core));
   TEST_ASSERT_EQUAL_INT32(SC_OK, ta_fetch_txn_with_uuid(&ta_core.cache, uuid, res));
   TEST_ASSERT_EQUAL_INT32(SENT, res->status);
-  tryte_t all[NUM_TRYTES_SERIALIZED_TRANSACTION + 1] = {};
-  flex_trit_t* transaction_flex_trits = transaction_serialize(res->txn);
-  flex_trits_to_trytes(all, NUM_TRYTES_SERIALIZED_TRANSACTION, transaction_flex_trits, NUM_TRITS_SERIALIZED_TRANSACTION,
-                       NUM_TRITS_SERIALIZED_TRANSACTION);
-  tryte_t trytes[NUM_TRYTES_MESSAGE + 1] = {};
-  flex_trits_to_trytes(trytes, NUM_TRYTES_TAG, transaction_tag(res->txn), NUM_TRITS_TAG, NUM_TRITS_TAG);
-  TEST_ASSERT_EQUAL_STRING(TEST_TAG, (char*)trytes);
-  flex_trits_to_trytes(trytes, NUM_TRYTES_HASH, transaction_address(res->txn), NUM_TRITS_HASH, NUM_TRITS_HASH);
-  TEST_ASSERT_EQUAL_STRING(TEST_ADDRESS, (char*)trytes);
-  flex_trits_to_trytes(trytes, NUM_TRYTES_MESSAGE, transaction_message(res->txn), NUM_TRITS_MESSAGE, NUM_TRITS_MESSAGE);
-  TEST_ASSERT_EQUAL_MEMORY(test_transfer_message, trytes, TEST_TRANSFER_MESSAGE_LEN);
-
-  ta_send_transfer_req_free(&req);
+
+  TEST_ASSERT_EQUAL_INT32(2, bundle_transactions_size(res->bundle));
+
+  for (int i = 0; i < txn_num_in_bundle; ++i) {
+    tryte_t all[NUM_TRYTES_SERIALIZED_TRANSACTION + 1] = {};
+    iota_transaction_t* txn = bundle_at(res->bundle, i);
+    flex_trit_t* transaction_flex_trits = transaction_serialize(txn);
+    flex_trits_to_trytes(all, NUM_TRYTES_SERIALIZED_TRANSACTION, transaction_flex_trits,
+                         NUM_TRITS_SERIALIZED_TRANSACTION, NUM_TRITS_SERIALIZED_TRANSACTION);
+    tryte_t trytes[NUM_TRYTES_MESSAGE + 1] = {};
+    flex_trits_to_trytes(trytes, NUM_TRYTES_TAG, transaction_tag(txn), NUM_TRITS_TAG, NUM_TRITS_TAG);
+    TEST_ASSERT_EQUAL_STRING(TEST_TAG, (char*)trytes);
+    flex_trits_to_trytes(trytes, NUM_TRYTES_HASH, transaction_address(txn), NUM_TRITS_HASH, NUM_TRITS_HASH);
+    TEST_ASSERT_EQUAL_STRING(TEST_ADDRESS, (char*)trytes);
+    flex_trits_to_trytes(trytes, NUM_TRYTES_MESSAGE, transaction_message(txn), NUM_TRITS_MESSAGE, NUM_TRITS_MESSAGE);
+    TEST_ASSERT_EQUAL_MEMORY(test_transfer_message[i], trytes, TEST_TRANSFER_MESSAGE_LEN);
+    free(transaction_flex_trits);
+  }
+
+  for (int i = 0; i < txn_num_in_bundle; i++) {
+    ta_send_transfer_req_free(&req[i]);
+    free(json[i]);
+  }
+
   hash_array_free(raw_txn_array);
   ta_fetch_txn_with_uuid_res_free(&res);
-  free(transaction_flex_trits);
-  free(json);
 }
 
 int main(int argc, char* argv[]) {
```

Comment:
This should be `int main(void)`.

Reply:
We need to use `argv` when setting the configures